app/components/listing/FeedbackModal.tsx

```diff
@@ -1,28 +0,0 @@
-import React from "react";
-import Modal from "react-modal";
-import { Organization, Service } from "../../models";
-import { FeedbackForm } from "./feedback/FeedbackForm";
-
-export const FeedbackModal = ({
-  organization,
-  service,
-  isOpen,
-  setIsOpen,
-}: {
-  isOpen: boolean;
-  setIsOpen: (val: boolean) => void;
-  organization: Organization;
-  service?: Service;
-}) => (
-  <Modal
-    isOpen={isOpen}
-    className="feedback__Modal"
-    overlayClassName="feedback__Overlay"
```

Comment:
These are CSS classes that are actually still defined in their corresponding CSS file. Could you remove the following lines here? https://github.com/ShelterTechSF/askdarcel-web/blob/f6db93aedb298a8a0d2d3069fed1a94b6bf7f5f4/app/components/listing/ActionBar.scss#L39-L71